include/z64item.h

```diff
@@ -7,10 +7,27 @@
 // TODO fill out these enums
 
 typedef enum {
-    /* 0x0 */ EQUIP_SWORD,
-    /* 0x1 */ EQUIP_SHIELD
+    /* 0x0 */ EQUIP_TYPE_SWORD,
+    /* 0x1 */ EQUIP_TYPE_SHIELD,
+    /* 0x2 */ EQUIP_TYPE_MAX
 } EquipmentType;
 
+typedef enum {
+    /* 0 */ EQUIP_VALUE_SWORD_NONE,
+    /* 1 */ EQUIP_VALUE_SWORD_KOKIRI,
+    /* 2 */ EQUIP_VALUE_SWORD_RAZOR,
+    /* 3 */ EQUIP_VALUE_SWORD_GILDED,
+    /* 4 */ EQUIP_VALUE_SWORD_DIETY,
+    /* 5 */ EQUIP_VALUE_SWORD_MAX,
+} EquipValueSword;
+
+typedef enum {
+    /* 0 */ EQUIP_VALUE_SHIELD_NONE,
+    /* 1 */ EQUIP_VALUE_SHIELD_HERO,
+    /* 2 */ EQUIP_VALUE_SHIELD_MIRROR,
+    /* 3 */ EQUIP_VALUE_SHIELD_MAX
+} EquipValueShield;
```

Comment:
I feel this set of enums are all a bit verbose, but I don't really think we can shorten them much without losing clarity

Reply:
In OoT, this name was chosen because there's `EQUIP_TYPE_XXX`, `EQUIP_INV_XXX`, and `EQUIP_VALUE_XXX` which all need distinguishing. I haven't introduced `EQUIP_INV_XXX` yet in MM although maybe it could be introduced? It's a mess.